shared/actions/login/index.js

```diff
@@ -429,7 +435,8 @@ function makeKex2IncomingMap (dispatch, getState) : incomingCallMapType {
               storeSecret: false
             }),
             onBack: () => dispatch(cancelLogin(response)),
-            error: retryLabel
+            error: retryLabel,
+            username
```

Comment:
It doesn't go into the store at all, the login flow avoids the store (as we've discussed previously)